### DISCUSSION TITLE
another user committing

### DISCUSSION THREAD

**Discussion by bugrahankaramollaoglu** [community/community#36164] [GitHub Education] [answered]
there seems to be another user named 'Tsunak' making commits to my repositories. Commits are mine but why is it uploaded via another username?
<img width="953" alt="Ekran Resmi 2022-10-14 ÖS 8 24 56" src="https://user-images.githubusercontent.com/85173065/195905810-01ed4489-c611-438b-af7a-78f1dbcf2a50.png">
<img width="343" alt="Ekran Resmi 2022-10-14 ÖS 8 25 05" src="https://user-images.githubusercontent.com/85173065/195905819-dc14f011-9b3d-4144-aaf4-c01a830226aa.png">

**Comment by FahdSeddik** [comment]
Hey @bugrahankaramollaoglu 
If you were using either github desktop or git to push your changes, please make sure you logout from all your accounts and re-install. I suspect that what's happening is that this is another account that you might've created by accident and would have access to the repositories. Hope this helps!

**Comment by radar-blip** [comment]
Not sure if this will help in your case, but there was a similar issue here: #29903
Possible that you and another user have the same email.

**Comment by jfullstackdev** [comment]
and the commits are not even verified, 

![image](https://user-images.githubusercontent.com/47092464/196042259-84af2864-c26d-4c2c-92b8-4d5b3bbf4346.png)

do you have the other one as your test user ? that will make a mess even if you are not using that account to commit those, I had the same experience in the past

**Comment by airtower-luna** [comment, marked as answer]
Which user a commit is assigned to depends on the author email address in the commit. I looked at two commits ([one that shows at yours](https://github.com/bugrahankaramollaoglu/42istanbul/commit/db4781d35312ba1ee8a62d5a7989305fe72a4ec4.patch) and [one that does not](https://github.com/bugrahankaramollaoglu/42istanbul/commit/97d4d12a61e76e535cecf73f7439ffe7b0d57899.patch)) and indeed the email addresses are different. Most likely the `user.email` setting on the computer where you made the commits is wrong.

You can set the correct email with:

```
git config --global user.email YOUR_EMAIL_HERE
```

The email address must be one that's on your account for the commits to count as yours.

**Reply by bugrahankaramollaoglu** [reply]
that seems to be the problem. thank you all countless times
